astropy/utils/iers/iers.py

```diff
@@ -95,15 +98,15 @@ class IERS(QTable):
     iers_table = None
 
     @classmethod
-    def open(cls, file=None, **kwargs):
+    def open(cls, file=None, cache=False, **kwargs):
```

Comment:
Sorry for the belated comment, but `cache` should be documented under `Parameters`. Maybe:
```
Parameters
---------------
file : str or None
    Full local or network path to the ascii file holding IERS data,
    for passing to the `read` class method.
cache : bool
    Whether to cache any files downloaded from the network.
    Default is `False` since the IERS files are regularly updated.
**kwargs
    Further optional arguments that are available for some IERS
    subclasses.
```